src/test/java/com/epam/tc/hw2/ex1/Exercise1.java

```diff
@@ -0,0 +1,86 @@
+package com.epam.tc.hw2.ex1;
+
+import com.epam.tc.hw2.SetUpAndTearDown;
+import java.util.Arrays;
+import java.util.List;
+import java.util.Locale;
+import org.assertj.core.api.Assertions;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.testng.annotations.Test;
+
+
+public class Exercise1 extends SetUpAndTearDown {
+
+    //Data
+    private String homeNavItem = "HOME";
+    private String contactNavItem = "CONTACT FORM";
+    private String serviceNavItem = "SERVICE";
+    private String metalsAndColorsNavItem = "METALS & COLORS";
```

Comment:
And how do declare string constant properly?

Reply:
Maybe private final